CHANGELOG.unreleased.md

```diff
@@ -6,6 +6,8 @@
 
 ### Bug fixes
 
+- [Xen servers] Handle pool UUID change [#4299](https://github.com/vatesfr/xen-orchestra/issues/4299) (PR [#4302](https://github.com/vatesfr/xen-orchestra/pull/4302))
```

Comment:
IMHO, this is unclear for users.

Users should be able to read the changelog and be able to say "oh yeah, I had this issue, I'm happy it's now fixed".